CHANGELOG.md

```diff
@@ -24,6 +24,28 @@ All notable changes to this project will be documented in this file, in reverse
 
 - Nothing.
 
+## 1.2.1 - 2019-12-10
```

Comment:
Please don't add a release date. Leave that to the person merging your PR and creating a release.

Reply:
thank you, new to this :)